packages/headless/src/controllers/core/facets/facet/headless-core-facet-options.ts

```diff
@@ -64,6 +64,51 @@ export interface FacetOptions {
   sortCriteria?: FacetSortCriterion;
 }
 
+export interface FacetOptionsV2 {
```

Comment:
There seems to be a lot of overlap with the existing option ? And not use in any place ?

If I understand the reasoning well, I'd say the correct way would be to define the new "limited" facet options in `controllers/product-listing` by using `Pick` or `Omit` from `FacetOptions`.

In any case, this definition should not exist in the `core` version of the controller